plugins/synapticsmst/synapticsmst-device.c

```diff
@@ -264,6 +264,7 @@ synapticsmst_device_read_board_id (SynapticsMSTDevice *device,
 				   guint8 *byte,
 				   GError **error)
 {
+	guint32 dwData[4] = {0};
```

Comment:
Shouldn't this be guint8 dwData[4] = {0} ?  You only need 4 bytes it seems.

Reply:
Fixed in latest push.